qemu/tests/cfg/virtio_scsi_cdrom.cfg

```diff
@@ -0,0 +1,92 @@
+- virtio_scsi_cdrom:
+    virt_test_type = qemu
+    type = unattended_install
+    backup_image = yes
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
+    shutdown_cleanly = yes
+    shutdown_cleanly_timeout = 120
+    force_create_image = yes
+    guest_port_unattended_install = 12323
+    kernel = vmlinuz
+    initrd = initrd.img
+    # Throw errors if guest screen is inactive
+    inactivity_watcher = error
+    # Inactivity treshold to error the test
+    inactivity_treshold = 1800
+    image_verify_bootable = no
+    # Backup images from nfs when install failed
```

Comment:
we don't have the nfs for this funtion any more.

Reply:
Thanks, have updated.